superset-frontend/src/components/Pagination/PaginationButton.types.ts

```diff
@@ -0,0 +1,23 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+export interface PaginationButton {
```

Comment:
I think you can just call this file `./components/Pagination/types.ts`. I haven't seen this filename pattern (`ComponentName.types.ts`) in Superset.

Maybe also call the type `PaginationButtonProps` to avoid being confused with it being an actual React component.

Reply:
I usually do this when it's just "Types". If you want to rename `PaginationButtonProps .ts` it is better.

Reply:
Creating a separate file for a single React props interface is not a pattern we've seen in Superset, and there are many more places that uses `types.ts` or a `types/` folder with larger files encompassing a set of related types.

I'd recommend follow the convention.